specs/src/state_machine_modules_v4.md

```diff
@@ -0,0 +1,35 @@
+# State Machine Modules v4
+
+The modules used in app version 4 are:
+
+## `celestia-app` modules
+
+- [blob](https://github.com/celestiaorg/celestia-app/blob/main/x/blob/README.md)
+- [minfee](https://github.com/celestiaorg/celestia-app/blob/main/x/minfee/README.md)
+- [mint](https://github.com/celestiaorg/celestia-app/blob/main/x/mint/README.md)
+- [signal](https://github.com/celestiaorg/celestia-app/blob/main/x/signal/README.md)
+- [tokenfilter](https://github.com/celestiaorg/celestia-app/blob/main/x/tokenfilter/README.md)
```

Comment:
Would recommend versioned links, idk if there's a markdown link checker workflow but they're notoriously annoying lol, especially with stuff that points at main branches

Reply:
69708806bc3bebb2cb231e612a5a22142d2b5f48